requirements.txt

```diff
@@ -7,4 +7,5 @@ skorch<0.14
 joblib
 mne
 einops
-docstring_inheritance
\ No newline at end of file
+docstring_inheritance
+torchinfo
```

Comment:
Can you add this to `environment.yml` and to the list in `setup.py` for the CI?

Reply:
done